client/reader/stream/index.jsx

```diff
@@ -632,7 +633,8 @@ class ReaderStream extends Component {
 					renderItem={ this.renderPost }
 					renderLoadingPlaceholders={ this.renderLoadingPlaceholders }
 					className="stream__list"
-					context={ this.state.listContext ?? false }
```

Comment:
Passing `false` as a fallback tells the `InfiniteStream` component that context is loaded before it actually is. `false` is evaluated by the `_contextLoaded` method of that component as having context loaded (as a way to say there is no context expected to load). Since this is passed as a fallback, it causes the infiniteStream to start acting like our context is loaded before it is.